.github/workflows/build.yml

```diff
@@ -3,24 +3,27 @@ on:
   pull_request:
   push:
     branches: [ master ]
+
 jobs:
   tests:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          submodules: recursive
       - uses: actions/setup-node@v2
-      - uses: cachix/install-nix-action@v13
-      - uses: cachix/cachix-action@v10
+      - uses: onbjerg/foundry-toolchain@v1
         with:
-          name: radicle-drips-hub
-          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+          version: nightly
+
       - name: install
-        run: nix-shell --run "make install"
+        run: make install
+
       - name: lint
-        run: nix-shell --run "make lint"
+        run: make lint 
+
       - name: test
         if: ${{ github.event_name == 'pull_request' }}
-        run: nix-shell --run "make test"
-      - name: test deep
-        if: ${{ github.event_name == 'push' }}
-        run: nix-shell --run "make test_deep"
+        env:
+          FOUNDRY_FUZZ_RUNS: 50000
+        run: make test
```

Comment:
Almost there, now the long fuzz runs are run for the PR CI runs and after merging to `master` nothing is run at all.